t/t3431-rebase-ignore-ws.sh

```diff
@@ -0,0 +1,70 @@
+#!/bin/sh
+#
+# Copyright (c) 2019 Rohit Ashiwal
+#
+
+test_description='git rebase --ignore-whitespace tests'
+
+. ./test-lib.sh
+
+cat >patch1.patch <<EOF
+diff --git a/main.c b/main.c
+new file mode 100644
+--- /dev/null
++++ b/main.c
+@@ -0,0 +1,6 @@
++#include <stdio.h>
++
```

Comment:
We can drop the `#include <stdio.h>` here, as it doesn't contribute anything to the outcome of the test.